genienlp/server.py

```diff
@@ -168,6 +188,10 @@ def parse_argv(parser):
     parser.add_argument('--stdin', action='store_true', help='Interact on stdin/stdout instead of TCP')
     parser.add_argument('--locale', default='en', help='locale tag of the language to parse')
 
+    # for confidence estimation:
+    parser.add_argument('--calibrator_path', type=str, default=None, help='If provided, will be used to output confidence scores for each prediction.')
+    parser.add_argument("--mc_dropout", action='store_true', help='Monte Carlo dropout')
+    parser.add_argument("--mc_dropout_num", type=int, default=0, help='Number of samples to use for Monte Carlo dropout')
```

Comment:
What is the correct configuration to use for MC dropout? Can we make it so the estimated confidence model stores how many MC dropout samples to take (or none if dropout features are not used)? Because Genie currently has no way to pass per-model flags to server.py

Reply:
Fixed this.